src/lib/user.ts

```diff
@@ -16,3 +19,15 @@ export const register = (login: string, password: string) => {
 export const info = (token: string) => {
   return infoBackend(token)
 }
+
+export const init2fa = (password: string) => {
```

Comment:
I'd skip `{` `}` and `return`

Reply:
Robiłem tak jak wyżej. Może lepiej tak zostawic